src/db/tables/subscribers.js

```diff
@@ -193,6 +193,27 @@ async function setAllEmailsToPrimary (subscriber, allEmailsToPrimary) {
 }
 /* c8 ignore stop */
 
+/**
+ * @param {import("knex/types/tables").SubscriberRow} subscriber
+ * @param {boolean} monthlyMonitorReport
+ */
+// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
+/* c8 ignore start */
+async function setMonthlyMonitorReport (subscriber, monthlyMonitorReport) {
+  const updated = await knex('subscribers')
+    .where('id', subscriber.id)
+    .update({
+      monthly_monitor_report: monthlyMonitorReport || false,
```

Comment:
If this parameter inadvertently doesn't get passed, we don't necessarily want to overwrite whatever's in there already, right?

```suggestion
      monthly_monitor_report: monthlyMonitorReport,
```